src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPageAzure.tsx

```diff
@@ -0,0 +1,403 @@
+    // Have to set it before the route change otherwise the previous title will be announced.
+    // It won't be reannounced on subsequent query param changes.
+    setPageTitle(`${defaultPageTitle} - Search results`);
+    await router.push(
+      {
+        pathname: '/find-statistics',
+        query: nextQuery as ParsedUrlQuery,
+      },
+      undefined,
+      {
+        shallow: true,
+      },
+    );
+  };
+
+  const handleSortBy = async (nextSortBy: PublicationSortOption) => {
+    await updateQueryParams({
+      ...omit(router.query, 'page'),
+      sortBy: nextSortBy,
+    });
+
+    logEvent({
+      category: 'Find statistics and data',
+      action: 'Publications sorted',
+      label: nextSortBy,
+    });
+  };
+
+  const handleChangeFilter = async ({
+    filterType,
+    nextValue,
+  }: {
+    filterType: PublicationFilter;
+    nextValue: string;
+  }) => {
+    const newParams =
+      nextValue === 'all'
+        ? omit(router.query, 'page', filterType)
+        : {
+            ...omit(router.query, 'page'),
+            [filterType]: nextValue,
+            sortBy,
+          };
+
+    await updateQueryParams(newParams);
+
+    logEvent({
+      category: 'Find statistics and data',
+      action: `Publications filtered by ${filterType}`,
+      label: nextValue,
+    });
+  };
+
+  const handleResetFilter = async ({
+    filterType,
+  }: {
+    filterType: PublicationFilter | 'all';
+  }) => {
+    // Reset sortBy to newest if removing a search and sorting by relevance
+    const nextSortBy = search && sortBy === 'relevance' ? 'newest' : sortBy;
+
+    const newParams =
+      filterType === 'all'
+        ? {
+            ...omit(router.query, 'page', ...publicationFilters),
+            sortBy: nextSortBy,
+          }
+        : {
+            ...omit(router.query, filterType, 'page'),
+            sortBy: nextSortBy,
+          };
+    await updateQueryParams(newParams);
+
+    logEvent({
+      category: 'Find statistics and data',
+      action: `Reset ${filterType} filter`,
+    });
+  };
+
+  const totalResultsMessage = `${totalResults} ${
+    totalResults !== 1 ? 'results' : 'result'
+  }`;
+
+  return (
+    <Page
+      title={defaultPageTitle}
+      // Don't include the default meta title when filtered to prevent too much screen reader noise.
+      includeDefaultMetaTitle={pageTitle === defaultPageTitle}
+      metaTitle={pageTitle}
+    >
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-full">
+          <p className="govuk-body-l">
+            Search and browse statistical summaries and download associated data
+            to help you understand and analyse our range of statistics.
+          </p>
+          <SearchForm
+            label="Search publications"
+            value={search}
+            onSubmit={nextValue =>
+              handleChangeFilter({ filterType: 'search', nextValue })
+            }
+          />
```

Comment:
The validation feels quite aggressive on this, I think the clear search button will help but we might want to alter the validation settings to onSubmit instead of onChange to avoid UI jumping around as you type